draft-enghardt-panrg-path-properties.md

```diff
@@ -46,8 +48,21 @@ This information is expressed as properties of paths between two endpoints.
 # Introduction
 
 Because the current Internet provides an IP-based best-effort bit pipe, endpoints have little information about paths to other endpoints.
-A Path Aware Network exposes information about one or multiple paths through the network to endpoints,
-so that endpoints can use this information.
+A Path Aware Network exposes information about one or multiple paths through the network to endpoints or the network infrastructure.
+
+It is impossible to provide an exhaustive list of path properties, as with every new technology and protocol, novel properties might become relevant.
+In order to specify broadly relevant path properties, scenarios in which knowledge about (additional) path information gives advantages are described here.
+
+Traffic policies leveraging path awareness can be defined.
+Such policies can allow or disallow sending traffic over specific networks or nodes, select an appropriate protocol depending on the capabilities of the on-path devices (e.g., using QUIC instead of TCP if no device blocks UDP traffic), or adjust protocol parameters to the used path.
+
+Network monitoring systems in a path aware network can associate their measurements with the actual path used, and observe if the measurements conform with the advertised path properties.
+Network operators can use the path information to specify comprehensive Quality of Service (QoS) guarantees.
+
+In the case that an entity can choose between a set of different paths, richer path information can lead to more optimal usage of networked resources.
+An AS can use path information as additional routing metric leading to more informed routing decisions.
```

Comment:
Here we mean on an AS level, right? Would that be path properties like "what ASes are further along the path", in a sense of "who is operating them", or something like this? If so, isn't this more about policy and less about efficient resource usage? Or are there other path properties that make sense of an AS level for resource usage, which are not covered by usual BGP attributes (local preference? or was that MED?)?
We should make sure that this use case clearly differs from the policy use case above.

Reply:
I guess for AS level path selection, the difference between path selection and traffic policy is very small, as the way ASes setup their policies is by selecting paths...

I guess for ASes (assuming they use BGP), "traffic policies" mean allowing and disallowing certain AS paths based on information from path properties about these ASes; and "path selection" means choosing the paths with the most benefits (e.g., lowest latency, highest maximum data rate, best load balancing, ...).
I think in BGP, regarding performance, we can only argue about intra-network links and links incoming and outgoing of the network, since we have little information about the rest of the path (list of AS numbers only).